### PR TITLE
rustfmt :- format the code.

### PR DESCRIPTION
**Pull request by naveensrinivasan**
- Included ignore based on this comment https://github.com/Lind-Project/safeposix-rust/pull/238#issuecomment-2103670963

## Description

Fixes # (issue)

 

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- Test A - `lind_project/tests/test_cases/test_a.c`
- Test B - `lind_project/tests/test_cases/test_b.c`

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to a pull request and/or merged in other modules (native-client, lind-glibc, lind-project)

### CODE REVIEW

**Review by JustinCappos — APPROVED** [review]
LGTM

**Review by Yaxuan-w — APPROVED** [review]
Changes primarily focus on standardizing the codebase. This standardization is not only beneficial for future development efforts but also facilitates quicker and easier understanding for new contributors. LGTM so approved.

File: .rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ignore = [
+    "src/safeposix/syscalls/net_constants.rs",
+    "src/safeposix/syscalls/sys_constants.rs",
+    "src/safeposix/syscalls/fs_constants.rs",
+]
```
Comment (rennergade) [inline .rustfmt.toml]: Sorry to be pedantic, should be a newline at the end of this file.

**Comment by rennergade** [comment]
This is great. I think this kind of consistency as we go forward as well as the automation should be incredibly useful. Thanks naveen!

I'm ready to merge once that newline is added.

**Comment by rennergade** [comment]
Great! Merged.

**Pull request merged**